sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java

```diff
@@ -65,7 +66,13 @@
 public class BigQueryIOJsonIT {
   private static final Logger LOG = LoggerFactory.getLogger(BigQueryIOJsonIT.class);
 
-  @Rule public final transient TestPipeline p = TestPipeline.create();
+  private static PipelineOptions testOptions = TestPipeline.testingPipelineOptions();
+
+  static {
+    testOptions.setTempLocation("gs://bigqueryio-json-it-temp/java-tmp");
```

Comment:
How do you expect this test to run for those who don't have access to `gs://bigqueryio-json-it-temp`? We have a bunch of tests that do something like `options.setTempLocation(options.getTempRoot() + "/java-tmp");` can you do something like that?

Reply:
Was going to make the bucket available for allUsers, but your suggestion is a lot more reasonable. 
Made the change.